corelib/constant.py

```diff
@@ -32,10 +32,10 @@
 char_dict_path = './data/text_reco/char_dict_en.json'
 ord_map_dict_path = './data/text_reco/ord_map_en.json'
 text_reco_url = 'models/crnn_syn90k_saved_model:predict'
-#text_detect_url = 'models/east:predict'
 text_detect_url = 'models/tbpp:predict'
 coco_names_path = './data/object_detect/coco.names'
 scene_labels_path = './data/scene_detect/places365.txt'
-#object_detect_url = 'models/yolov3:predict'
 object_detect_url = 'models/efficientdet:predict'
 scene_detect_url = 'models/places:predict'
+#text_detect_url = 'models/east:predict'
```

Comment:
There are two text detect apis here and two object detection apis
 Create an issue where we can allow users to sepecf which api they want to use. Need not work on it right away.

Reply:
Added the issue #209